core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -96,8 +102,8 @@ export class BubbleCluster extends Chart {
         .attr("values", filter.values);
   }
 
-  _addNewBubbleGroups(leaves) {
```

Comment:
Renaming all occurrences of `leaves` to `nodes` wasn't strictly necessary, but I wanted to avoid confusion - previously these _were_ always leaves (even though the functions never cared I guess), but now they are not.